### DISCUSSION TITLE
Highlights doens't work

### DISCUSSION THREAD

**Discussion by Hirashi3630** [jdanielmourao/obsidian-sanctum#196] [Q&A]
Hi, I can't find anyone asking this before, so here I go.

In the wiki, it's shown that highlights display as an underline + animated on hover

![working multicolored highlights](https://user-images.githubusercontent.com/91087143/139734042-ea7ed245-9cee-4b18-8466-4a1830bf3f69.gif)

mine looks like this (static)

![mine multicolored highlights](https://user-images.githubusercontent.com/37778278/194301866-8c45ab8b-7441-435f-8052-c81a7c02e258.png)

Is this something I need to turn on?

I've searched in Style Settings plugin and found nothing 
(I didn't change anything in settings + Contextual Typography is installed too if that matters)

Thanks

**Comment by robertdcooper** [comment]
I was curious about this as well. Did you figure it out?

**Comment by Hirashi3630** [comment]
> I was curious about this as well. Did you figure it out?

Hi, never figured it out.
In the newer versions the static looks much better but sadly still not animated.
